resources/public/api/conf/3.0/schemas/retrieve_self_assessment_charge_history_response.json

```diff
@@ -48,6 +48,11 @@
             "description": "The reason for the change.",
             "type": "string",
             "example": " Amended return, Customer Request"
+          },
+          "poaAdjustmentReason": {
+            "description": "A numeric code indicating the reason for adjustment to Payments on Account.\n\nThe possible values (with their meanings) are:\n\n001: My business profits are down/my business has ceased\n\n002: Other income has gone down\n\n003: Tax Allowances and reliefs have gone up\n\n004: Tax deducted at source is more than previous year.\n\n005: Increasing the PoA after a prior adjustment",
```

Comment:
Has this description been reviewed?

- 'My business' is odd.
- 'My business profits are down/my business has ceased' -- is the slash an 'or'?
- Some points end with period, some don't.